docs/notebooks/nb_06_integrals.ipynb

```diff
@@ -0,0 +1,171 @@
+{
```

Comment:
<p>Same here! Some equations are not displaying correctly</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/evangelistalab/forte/pull/246/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>